.github/workflows/end-to-end-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: "end-to-end_tests"
+on:
+    push:
+        branches: [develop]
+    pull_request:
+        branches: [develop]
+jobs:
+    "end-to-end_tests":
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                shell: bash
```

Comment:
Looks like this is [the default](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell), is it specified here for clarity?